pkg/svcat/service-catalog/instance.go

```diff
@@ -25,13 +25,13 @@ import (
 	"github.com/pkg/errors"
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	"k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/fields"
+	"k8s.io/apimachinery/pkg/labels"
 	"k8s.io/apimachinery/pkg/util/wait"
 )
 
 const (
 	// FieldServicePlanRef is the jsonpath to an instance's plan name (Kubernetes name).
-	FieldServicePlanRef = "spec.clusterServicePlanRef.name"
+	FieldServicePlanRef = v1beta1.GroupName + "/spec.clusterServicePlanRef.name"
```

Comment:
FieldServicePlanRef -> LabelServicePlanRef ??

Reply:
I removed this constant and replaced it with equivalent from v1beta package